docs/deployment_ops_guide.md

```diff
@@ -0,0 +1,860 @@
+
+```
+# mainnet lily build
+$ make mainnet
+
+# calibnet lily build
+$ make calibnet
+```
+
+#### Docker image builds
+
+Lily docker images can be produced for multiple networks via `make` by passing one of the following targets as an argument: `docker-mainnet`, `docker-calibnet`, `docker-nerpanet`, `docker-butterflynet`, `docker-interopnet`, `docker-2k`.
+
+### Using Docker to run service dependencies
+
+Once your local Lily build is ready, you can get supporting services up and running quickly using pre-built Docker containers with `docker-compose`.
+
+This is useful for testing and seeing how things work, but may not be suited to a production environment (with a Lily-mainnet node), which we recommend planning according to your needs and the tasks you will run.
+> _(Note: `docker`, `docker-compose`, and `make` are dependencies. See [Docker](https://docs.docker.com/engine/install/) and [GNU Make](https://www.gnu.org/software/make/) documentation for installation instructions on your system.)_
+
+Included in the [`docker-compose.yml`](https://github.com/filecoin-project/lily/tree/master/docker-compose.yml) are all the complimentary services that Lily requires for local debugging and development. These services come preconfigured to work with Lily's default ports.
+
+- TimescaleDB
+- Prometheus
+- Grafana
+- Jaeger Tracing (all-in-one)
+
+Once these services are started, you can build and initialize Lily. Here is how you can manage all of these services together using `docker-compose`:
+
+```
+# create and start services, then follow logs
+$ docker-compose up -d && docker-compose logs -f
+
+# create, start, and follow logs (interactively)
+$ docker-compose up
+
+# stop and destroy services
+$ docker-compose down
+
+# stop (but not destroy) services
+$ docker-compose stop
+
+# manage just the timescale service
+$ docker-compose start timescaledb
+$ docker-compose restart timescaledb
+$ docker-compose stop timescaledb
+```
+
+The services in the `docker-compose.yml` are defined as follows:
+
+- timescaledb
+- prometheus
+- grafana
+- jaeger
+- lily
+
+### Using Docker to run Lily
+
+The following steps will get a copy of Lily running locally using pre-built images found at [https://hub.docker.com/repository/docker/filecoin/lily](https://hub.docker.com/repository/docker/filecoin/lily). 
+
+1. Edit `docker-compose.yml` and uncomment the section describing `lily`.
+
+```
+# uncomment the following in docker-compose.yml
+...
+  lily:
+    container_name: lily
+    image: filecoin/lily:v0.8.0-calibnet
+    env_file:
+      - ./build/lily/docker.env
+    depends_on:
+      - prometheus
+      - timescaledb
+      - jaeger
+    ports:
+      - 1234:1234
+    volumes:
+      - lily_data:/var/lib/lily
+      - ./build/lily/docker_init.sh:/usr/bin/docker_init.sh
+    entrypoint: /usr/bin/docker_init.sh
+    command:
+      - daemon
+...
```

Comment:
There is no such section in the compose file (master). `docker.env` and `docker_init.sh` seem to be no longer around.